bedrock/settings/static_media.py

```diff
@@ -394,6 +394,26 @@
         ),
         'output_filename': 'css/firefox_new-bundle.css',
     },
+    'firefox_new_scene1': {
+        'source_filenames': (
+            'css/sandstone/sandstone-resp.less',
```

Comment:
Would it be worth splitting these out into common CSS bundles? Could we potentially get faster page loads on scene 2 if the common bundle was already cached from scene1?